resources/windows/wix/license/LICENSE.rtf

```diff
@@ -0,0 +1,375 @@
+{\rtf1\ansi\ansicpg1252\deff0{\fonttbl{\f0\fswiss\fcharset0 ArialMT;}{\f1\fnil\fcharset0 Courier New;}{\f2\fnil\fcharset0 Arial;}}
+{\colortbl ;\red0\green0\blue255;\red57\green57\blue57;}
+{\*\generator Msftedit 5.41.21.2510;}\viewkind4\uc1\pard\lang9\b\f0\fs28 License Summary:\b0\fs20\par
+\par
+\pard\fi-360\li360 Puppet - Apache 2.0\par
+Facter - Apache 2.0\par

```

Comment:
https://github.com/puppetlabs/vanagon/pull/286 was what I've done for this. Beyond that, I had just added licenses to components. I have no idea what the best way to deal with the actual content of the licenses below.